index.js

```diff
@@ -187,6 +187,9 @@ function use (extensions) {
     this.on('close', onfinalize)
     this.on('end', onfinalize)
     this.on('finish', this.finalize)
+    this.on('error', function (err) {
+      debug('stream error', err)
+    })
```

Comment:
Hm that feels like fighting the system. Why not leave this one event to the module consumer to handle? If they handle it, they know it happened, if don't, node will tell them.